client/dashboard/me/preferences-new-hosting-dashboard/index.tsx

```diff
@@ -51,6 +52,7 @@ const fields: Field< OptInFormData >[] = [
 ];
 
 export default function PreferencesLanguageForm() {
```

Comment:
I just noticed this bit of copy pasta that I left behind when I made this component 🙊 

```suggestion
export default function PreferencesOptInForm() {
```